src/Fragment.js

```diff
@@ -13,7 +13,7 @@ export default function Fragment() {
   const [reward, setReward] = useState(0)
   const [rowColor, setRowColor] = useState([...Array(10)].map(() => validTextColor.slice()))
 
-  var headerList = [
+  let headerList = [
```

Comment:
this is never reasigned, so better set a const. For variables that are never reasigned, better set a const